README.md

```diff
@@ -49,6 +49,10 @@ http {
   # hold your certificate data. 1MB of storage holds certificates for
   # approximately 100 separate domains.
   lua_shared_dict auto_ssl 1m;
+  # The "auto_ssl" shared dict is used to temporarily store various settings
+  # like the secret used by the hook server on port 8999. Do not change or
+  # omit it.
+  lua_shared_dict auto_ssl_settings 1m;
```

Comment:
I might be inclined to change this recommended size down to something like `64k`. While nobody's probably going to miss 1MB of RAM on a modern server, since this settings dict is only storing a single secret token (64 *bytes* long) and a single boolean value, even 64KB gives us plenty of space in case we want to store other things in here.